src/Inquirer/Console/ConsoleManager.cs

```diff
@@ -24,6 +24,21 @@ public int[] Render(string[] content, string[] bottomContent)
             return new [] { content.Length, bottomContent.Length };
         }
 
+        public int[] Render(ConsoleMessage[] content, ConsoleMessage[] bottomContent)
+        {
+            content.ToList().ForEach(item =>
+            {
+                if (item.ConsoleColor.HasValue)
+                    console.ForegroundColor = item.ConsoleColor.Value;
+
+                console.WriteLine(item.Message);
+            });
+            Newline();
+            console.ResetColor();
+            bottomContent.ToList().ForEach(item => console.WriteLine(item.Message));
```

Comment:
Since the `bottomContent` is also of the type `ConsoleMessage`, we should check the Color the same way that you have done for the content.